pkg/controller/kieapp/defaults/defaults_test.go

```diff
@@ -7276,11 +7276,11 @@ func TestKieExecutorMDB(t *testing.T) {
 	env, err := GetEnvironment(cr, test.MockService())
 	assert.Nil(t, err, "Error getting TestKieExecutorMDB environment")
 
-	assert.NotNil(t, cr.Status.Applied.Objects.Servers[0].MDBMaxSession)
+	assert.NotNil(t, cr.Status.Applied.Objects.Servers[0].KieExecutorMDBMaxSession)
 	mdbMaxSessionPassed := false
 	for _, env := range env.Servers[0].DeploymentConfigs[0].Spec.Template.Spec.Containers[0].Env {
-		if strings.HasPrefix(env.Name, "JBOSS_MDB") {
-			if env.Name != "JBOSS_MDB_MAX_SESSION" && env.Value == "40" {
+		if strings.HasPrefix(env.Name, "KIE_EXECUTOR_MDB") {
```

Comment:
there is a better way to test env, e.g.:
```
assert.Equal(t, "auth", getEnvVariable(env.Console.DeploymentConfigs[0].Spec.Template.Spec.Containers[0], "APPFORMER_INFINISPAN_SASL_QOP"))
```

Reply:
fixed